### ISSUE TITLE
3DS results not recorded in order notes when intent is Authorize (1684)

### ISSUE THREAD

**Issue by InpsydeNiklas** [woocommerce/woocommerce-paypal-payments#1799]
### Describe the Bug

For Advanced Card Processing payments, certain 3D Secure information is saved in the WooCommerce order notes. However, this behavior only happens when the intent is set to "Capture". When the intent is set to "Authorize", the 3D Secure order notes aren't created.
In this case, the data is only accessible from the PayPal order page linked in the transaction id.

### To Reproduce

1. Go to 'Advanced Card Processing' settings.
2. Enable '3D Secure'.
3. Set intent to 'Capture'.
4. Make a payment using a 3D Secure card: https://developer.paypal.com/docs/checkout/advanced/customize/3d-secure/test/
5. Visit the respective order.
6. Notice the 3D Secure information displayed as an order note.
7. Now, set intent to 'Authorize'.
8. Repeat the payment with a 3D Secure card.
9. Visit the order again.
10. See the absence of 3D Secure information in the order notes.

### Screenshots
When the data was successfully added to the order notes:
![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70433191/0764eb48-1851-42f5-b47f-6e7b5d4484b9)

### Expected Behavior
3D Secure information should be displayed in the order notes even when the intent is set to "Authorize".

### Actual Behavior
3D Secure information is only displayed in the order notes when the intent is set to "Capture".

**Comment by deisouss** [comment]
In addition to the CVV2 info, can you also pass the **Card Type** to the notes?

![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70108949/b37746ae-8bde-45d3-b96b-415e8df9a2f9)

This information is displayed in the PayPal section of the CVV2, thank you

**Comment by deisouss** [comment]
This was marked as completed but we are not seeing any information about the AVS results on the "Order Notes", has this change been pushed or do we need to do anything from our side to start seeing it?

![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70108949/7a546f9f-e3b1-4c79-bf5b-f1d810425764)

**Comment by InpsydeNiklas** [comment]
Hello @deisouss, this change will be part of the next update later this month. Until a pre-release version is available for testing, you can give it a try with [this test version](https://drive.google.com/uc?export=download&id=1i5yThvnlbnUxGZOYgrr-hGl0INtvLEQ7). There may still be more changes, and this test version has yet to go through the entire test cycle, so we recommend using it only for testing in a staging environment.

Beyond showing the fraud detection data for authorized orders, it now also prints the 3DS result:
![image](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70433191/66d02652-c55d-42ad-800f-1e0a06a805f1)
This data is stored in the order meta in the `_ppcp_paypal_fraud_result` & `_ppcp_paypal_3DS_auth_result` keys with the `woocommerce_paypal_payments_thee_d_secure_added` & `woocommerce_paypal_payments_fraud_result_added` actions firing after the respective data was added.

**Comment by deisouss** [comment]
hi @InpsydeNiklas the verification is now being added, thank you. Unfortunately there are some discrepancies and I am not sure exactly where the disconnect is happening.

1. On the WooCommerce notes the address verification does not reflect PayPal's verification. I checked on 3 orders and they all yield the same results.

Woo: 
    Address Match: N
    Postal Match: N

PayPal:
    Address Verification Service (AVS): **Exact match**

2. I logged in to manager.paypal.com and I found something interesting, the AVS response shows "N", just like what was posted to Woo, even though PayPal says exact match:
   AVS Street Match: 	N
   AVS Zip Match : 	
   
3. From what I can see, it seems that at **manager.paypal.com** the billing address is completely missing, which could have triggered the AVS response, but in paypal.com has the **correct AVS**. 

I am attaching screenshots. On a good note, the credit card type is being passed correctly

![Screenshot 2024-03-04 120248](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70108949/ce1837b7-fd4d-4f8f-aa0f-b93719fe239b)
![Screenshot 2024-03-04 120238](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70108949/9fee1773-ea68-4c56-a448-71edfdb96ca4)
![Screenshot 2024-03-04 120231](https://github.com/woocommerce/woocommerce-paypal-payments/assets/70108949/1d1306d8-3c8d-4d41-98a4-70498d0fd5e9)
